### PR TITLE
Feature/micro entity accounts

### PR DESCRIPTION
**Pull request by danfalc**

### CODE REVIEW

File: cypress/support/page_objects/MicroEntityBalanceSheet.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        cy.get('#fixedAssetsExpand').click();
+        cy.get('#currentAssetsExpand').click();
+        cy.get('#capitalAndReservesExpand').click();
+        cy.get('#otherNotesExpand').click();
+        cy.get('#abbreviatedNotesExpand').click();
+        return this;
+    }
+
+    openAndCheckAdditionalNotes() {
+        // Open each of the additional not screens in turn and conduct accessibility checks
```
Comment (rsuller) [inline cypress/support/page_objects/MicroEntityBalanceSheet.js]: not or note?

File: cypress/integration/accounts/micro_entity_accounts.spec.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+const submissionConfirmation = new SubmissionConfirmationPage();
+
+describe('File Company Accounts', () => {
+    beforeEach('Select Accounts from overview', () => {
+        // Select accounts from overview
+        companyOverview.selectLinkWithText("File company accounts");
+        cy.accessibilityCheck();
+    })
+
+    it('Successfully file Micro-entity accounts', () => {
```
Comment (rsuller) [inline cypress/integration/accounts/micro_entity_accounts.spec.js]: Can you check this test, please? It appears to have failed the last couple of items.

File: cypress/support/page_objects/MicroEntityBalanceSheet.js (hunk 1 above)
Comment (danfalc) [reply cypress/support/page_objects/MicroEntityBalanceSheet.js]: Note. Corrected 👍

File: cypress/integration/accounts/micro_entity_accounts.spec.js (hunk 2 above)
Comment (danfalc) [reply cypress/integration/accounts/micro_entity_accounts.spec.js]: I have taken out an unnecessary method and changed an erroneous id. Now the test fails but for accessibility reasons only

**Pull request merged**